Terminal.Gui/ConsoleDrivers/Net/NetDriver.cs

```diff
@@ -0,0 +1,902 @@
+//
+// NetDriver.cs: The System.Console-based .NET driver, works on Windows and Unix, but is not particularly efficient.
+//
+
+using static Terminal.Gui.ConsoleDrivers.ConsoleKeyMapping;
+using static Terminal.Gui.ConsoleDrivers.Net.NetEvents;
+
+namespace Terminal.Gui.ConsoleDrivers.Net;
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using static EscSeqUtils;
+
+internal sealed class NetDriver : ConsoleDriver
+{
+    private const int COLOR_BLACK = 30;
```

Comment:
Let's use `AnsiColorCode` for this since it now exists:  `.\Drawing\AnsiColorCode.cs`

Reply:
Good thought.

How about I do that when I get to NetDriver, specifically, though, which will probably be the next branch after the one I'm working on right now branched from this one?

That was just relocated code. No actual change.

Plus, it kinda looks like it shouldn't even be in that driver in the first place, but rather...I dunno... _somewhere_ else?